### PR TITLE
Nc fix/checkbox

### PR DESCRIPTION
**Pull request by reenphygeorge**
## Change Summary

* Default value input padding fixed

## Change type

- [x] feat: (new feature for the user, not a new feature for build script)
- [x] fix: (bug fix for the user, not a fix to a build script)
- [ ] docs: (changes to the documentation)
- [ ] style: (formatting, missing semi colons, etc; no production code change)
- [ ] refactor: (refactoring production code, eg. renaming a variable)
- [ ] test: (adding missing tests, refactoring tests; no production code change)
- [ ] chore: (updating grunt tasks etc; no production code change)

### CODE REVIEW

File: packages/nc-gui/components/general/ColorPicker.vue [hunk 1]
```diff
@@ -46,34 +43,18 @@ watch(picked, (n, _o) => {
...
-        </GeneralTooltip>
-      </button>
-    </div>
-
-    <a-card v-if="props.advanced" class="w-full mt-2" :body-style="{ padding: '0px' }" :bordered="false">
-      <div v-if="isPickerOn" class="flex justify-center">
-        <LazyGeneralChromeWrapper v-model="picked" class="!w-full !shadow-none" />
-      </div>
-    </a-card>
+  <div class="!w-20" v-for="colId in Math.ceil(props.colors.length / props.rowSize)" :key="colId">
```
Comment (wingkwong) [inline packages/nc-gui/components/general/ColorPicker.vue:46]: don't think `!` is required at all.

**Review by wingkwong — CHANGES_REQUESTED** [review]
image

**Review by wingkwong — COMMENTED** [review]
Can you provide a screenshot for verification? None of my previous comments are resolved.

**Comment by reenphygeorge** [comment]
Seems like ``:class="`bg-[${color}]`"`` is not working
Screenshot_20230927_132144

**Comment by wingkwong** [comment]
> Seems like `` :class="`bg-[${color}]`" `` is not working

Now it works on my local. Can you change it back and test once in your local?

image

**Review by wingkwong — COMMENTED** [review]
Also. I think the color block can be center-aligned. Besides, the tick in default value seems not align with the one in icon. May add some padding left.

image

**Comment by reenphygeorge** [comment]
> > Seems like `` :class="`bg-[${color}]`" `` is not working
> 
> Now it works on my local. Can you change it back and test once in your local?
> 
> image

With ``:class="`bg-[${color}]`"`` it sometimes works sometimes doesn't.

**Review by dstala — APPROVED** [review]

**Pull request merged**